bin/db-update.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+CALLEDPATH=`dirname $0`
+
+# Convert to an absolute path if necessary
+case "$CALLEDPATH" in
+  .*)
+    CALLEDPATH="$PWD/$CALLEDPATH"
+    ;;
+esac
+
+function _mysql_vars() {
+  if [ -z $DBPASS ]; then # password still empty
+    PASSWDSECTION=""
+  else
+    PASSWDSECTION="-p$DBPASS"
+  fi
+
+  HOSTSECTTION=""
+  if [ ! -z "$DBHOST" ]; then
+    HOSTSECTION="-h $DBHOST"
+  fi
+
+  PORTSECTION=""
+  if [ ! -z "$DBPORT" ]; then
+    PORTSECTION="-P $DBPORT"
+  fi
+}
+
+function mysql_cmd() {
+  _mysql_vars
+  echo "mysql -u$DBUSER $PASSWDSECTION $HOSTSECTION $PORTSECTION $DBARGS $DBNAME"
+}
+
+function mysqldump_cmd() {
+  _mysql_vars
+  echo "mysqldump -u$DBUSER $PASSWDSECTION $HOSTSECTION $PORTSECTION $DBARGS"
+}
+
+
+source "$CALLEDPATH/setup.conf"
+
+MYSQLCMD=$(mysql_cmd)
+MYSQLDUMP=$(mysqldump_cmd)
+
+echo "Creating mysqldump of drupal database and dumping at $PWD/$PRODDBNAME.mysql"
+$MYSQLDUMP  $PRODDBNAME > "$PRODDBNAME.mysql"
+sed -i 's/DEFINER=[^*]*\*/\*/g' "$PRODDBNAME.mysql"
+
+
+echo "Creating mysqldump of civicrm database and dumping at $PWD/$PRODCIVIDBNAME.mysql"
+$MYSQLDUMP $PRODDBNAME > "$PRODCIVIDBNAME.mysql"
```

Comment:
This variable looks incorrect

Reply:
thanks @seamuslee001 for spotting that error. Fixed it now.